apps/stats/management/commands/__init__.py

```diff
@@ -92,3 +101,24 @@ def get_date_from_file(filepath, sep):
             return line.split(sep)[0]
         except IndexError:
             return None
+
+
+def stats_from_model(model):
+    """Return the stats from the model ready to write to a file."""
+    data = model_to_dict(model)
+    del data['date']  # Already in the filename.
+    del data['addon']  # Already in the file's folder name.
+    del data['id']  # No need for the model's ID at all (eg: UpdateCount).
+    return json.dumps(data)
```

Comment:
I think I'm alright with cleanup up the `id` but if one wants to write some custom stuff with the returned json data I feel like having the `addon` and `date` or even the `id` in there could make sense.